test/integration/033_event_tracking_test/test_events.py

```diff
@@ -9,6 +10,7 @@
 import dbt.tracking
 import dbt.utils
 
+pytestmark = pytest.mark.skip("Experimental parser generates events non-deterministically, skipping.")
```

Comment:
Does this skip all the tests in this file? I don't see `pytestmark` being used anywhere so I'm not sure how that works

Reply:
That is correct, it marks all tests in the module as skipped!

https://docs.pytest.org/en/stable/reference.html#globalvar-pytestmark

Reply:
Well, I thought it did. https://github.com/dbt-labs/dbt/pull/3604/checks?check_run_id=3124252846